server/web/router.go

```diff
@@ -742,7 +778,16 @@ func (p *ControllerRegister) serveHttp(ctx *beecontext.Context) {
 	}
 
 	// session init
-	if p.cfg.WebConfig.Session.SessionOn {
+	currentSessionOn = BConfig.WebConfig.Session.SessionOn
+	originRouterInfo, originFindRouter = p.FindRouter(ctx)
+	if originFindRouter {
+		if !currentSessionOn && originRouterInfo.sessionOn {
```

Comment:
could we calcaulte the value of `sessionOn` when we create this route?

Reply:
Good idea. I'll try it.